pkg/server/server.go

```diff
@@ -389,8 +392,15 @@ func (s *Server) HTTPHandler() http.Handler {
 	handle("/api/console/knative-channels", authHandler(s.handleKnativeChannelCRDs))
 	handle("/api/console/version", authHandler(s.versionHandler))
 
+	// Get running kubernetes version and inject into helm handlers.
+	kubeVersion, err := actions.GetKubeVersion(s.KubeAPIServerURL, s.K8sClient.Transport)
+	if err != nil {
+		klog.Warningf("Failed to get server version %s", err.Error())
```

Comment:
how about "Failed to get cluster k8s version"?

Reply:
done